datadog_checks_downloader/datadog_checks/downloader/download.py

```diff
@@ -317,6 +317,11 @@ def get_wheel_relpath(self, standard_distribution_name, version=None, ignore_pyt
             raise MissingVersions(standard_distribution_name)
 
         if not version:
+            # Go through all wheels and remove alphas, betas and rcs
+            non_official_suffixes = ["a", "b", "rc"]
```

Comment:
Python's packaging provides Version that could be used in this case:

https://github.com/pypa/packaging/blob/5d7f0200b4cedf74da1a28a5c32d219e4e5f5252/src/packaging/version.py#L381-L396